examples/picorv32_ram/picorv32_ram.py

```diff
@@ -0,0 +1,60 @@
+
+import importlib
+import os
+import siliconcompiler
+
+def build_top():
+    # Core settings.
+    design = 'picorv32_top'
+    target = 'skywater130_demo'
+    die_w = 1000
+    die_h = 1200
+
+    # Create Chip object.
+    chip = siliconcompiler.Chip(design)
+
+    # Set default Skywater130 PDK / standard cell lib / flow.
+    target_module = importlib.import_module(f'targets.{target}')
+    chip.use(target_module)
+
+    # Set design source files.
+    rootdir = os.path.dirname(__file__)
+    chip.input(os.path.join(rootdir, f"{design}.v"))
+    chip.input(os.path.join(rootdir, "picorv32.v"))
+    chip.input(os.path.join(rootdir, "sky130_sram_2k.bb.v"))
+    chip.input(os.path.join(rootdir, f"{design}.sdc"))
```

Comment:
Is the `rootdir` addition meant to facilitate running this from other directories? It shouldn't be necessary if the user runs `picorv32_ram.py` from the directory it currently resides within.

Reply:
Yeah, and for tests. Given the choice, I don't think it's ideal to have the build results depend on the current working directory, but plenty of build tools like Make and CMake act like that by default.